content/extensions/ui-extensions.md

```diff
@@ -0,0 +1,468 @@
+   ```
+
+**4.  Your API Token:**
+- Make the API call to retrieve your `API token` using the environment variables:
+   ```bash [snippet:Shell]
+        curl --request POST "$TARGET_PIM_URL/api/oauth/v1/token" \
+    --header "Content-Type: application/json" \
+    --header "Authorization: Basic $BASE64_ENCODED_CLIENTID_AND_SECRET" \
+    --data-raw '{
+    "grant_type": "password",
+    "username": "'"$API_USERNAME"'",
+    "password": "'"$API_PASSWORD"'"
+    }'
+   ```
+After retrieving the API token, store the `access_token` from the response in an environment variable:
+   ```bash [snippet:Shell]
+        export PIM_API_TOKEN="..."
+    // Replace with the actual token from the response
+   ```
+
+::: info
+ℹ️ Note that the token has a lifespan of one hour.
+:::
+
+::: info
+🛠 Custom apps are also supported. To use one, add a variable `app_access_token` with your API Token.
+:::
+
+#### 2. Create a UI extension
+You can create a UI extension once you have a valid PIM API token.
+
+```bash [snippet:Shell]
+    curl --request POST "$TARGET_PIM_URL/api/rest/v1/ui-extensions" \
+--header "Authorization: Bearer $PIM_API_TOKEN" \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "name": "my_awesome_extension",
+    "version": "V1.02",
+    "type": "link",
+    "position": "pim.product.header",
+    "configuration": {
+        "url": "https://myapp.com/product",
+        "default_label": "My awesome extension",
+        "labels": {
+          "en_US": "My awesome extension"
+        }
+    }
+}'
+```
+
+#### 3. Update a UI extension
+To update a UI extension, you must possess a valid PIM API token and the UUID of the extension you want to update.
+
+```bash [snippet:Shell]
+    curl --request PATCH "$TARGET_PIM_URL/api/rest/v1/ui-extensions/$EXTENSION_UUID" \
+--header "Authorization: Bearer $PIM_API_TOKEN" \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "version": "V1.03",
+    "type": "iframe",
+    "position": "pim.product.tab"
+}'
+```
+
+#### 4. Delete a UI extension
+To delete a UI extension, you must possess a valid PIM API token and the UUID of the extension you want to delete.
+
+```bash [snippet:Shell]
+    curl --request DELETE "$TARGET_PIM_URL/api/rest/v1/ui-extensions/$EXTENSION_UUID" \
+--header "Authorization: Bearer $PIM_API_TOKEN"
+```
+
+## Concepts
+
+### Type
+
+UI extensions are categorized by type, which determines their capabilities. Select the type that best suits your requirements:
++ action
++ iframe
++ link
+
+#### Link
+A **link** UI extension is crafted to open your external content in a new tab.
+
+#### Iframe
+An **iframe** UI extension allows to open your external content inside the PIM thanks to an iframe.
+
+An iframe (inline frame) is an HTML element that allows you to embed another HTML document within the current document. It is commonly used to display content from another source, such as a webpage, video, or interactive content, without leaving the current page.
```

Comment:
Less is better: merge these two sentences ;)

Reply:
The first is about the UI extension type, the second is about the HTML iframe. What is you proposition ?